### PR TITLE
btrfs to ext4 docs fixes

### PR DESCRIPTION
**Pull request by ionave**

### CODE REVIEW

File: running-coreos/bare-metal/installing-to-disk/index.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ coreos-install -d /dev/sda -C beta -c ~/cloud-config.yaml
 
 ## Manual Tweaks
 
-If cloud config doesn't handle something you need to do or you just want to take a look at the root btrfs filesystem before booting your new install just mount the ninth partition:
+If cloud config doesn't handle something you need to do or you just want to take a look at the root filesystem before booting your new install just mount the ninth partition:
 
 ```sh
 mount -o subvol=root /dev/sda9 /mnt/
```
Comment (marineam) [inline running-coreos/bare-metal/installing-to-disk/index.md]: `subvol=root` is only for btrfs so that should be dropped too.

**Comment by marineam** [comment]
just one issue with subvol, the rest looks good to me.

**Comment by ionave** [comment]
Alright, I've fixed this!

**Comment by marineam** [comment]
Please squash the commits together before merging. git rebase -i can help with that.

**Comment by ionave** [comment]
Ah. Okay. I think I've got it now :)

**Comment by marineam** [comment]
LGTM

**Pull request merged**